examples/guestbook/README.md

```diff
@@ -373,66 +408,90 @@ if (isset($_GET['cmd']) === true) {
 } ?>
 ```
 
-### Step Six: Create the guestbook service.
-
-Just like the others, you want a service to group your frontend pods.
-The service is described in the file `examples/guestbook/frontend-service.json`:
-
-```js
-{
-   "kind":"Service",
-   "apiVersion":"v1",
-   "metadata":{
-      "name":"frontend",
-      "labels":{
-         "name":"frontend"
-      }
-   },
-   "spec":{
-      "ports": [
-        {
-          "port":80
-        }
-      ],
-      "selector":{
-         "name":"frontend"
-      }
-   }
-}
+Note the use of the `redis-master` and `redis-slave` host names-- we're finding those services via the Kubernetes cluster's DNS service, as discussed above.
+
+### Step Six: Set up the guestbook frontend service.
+
+As with the other pods, we now want to create a service to group your frontend pods.
+The service is described in the file `examples/guestbook/frontend-service.yaml`:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    name: frontend
+  name: frontend
+spec:
+  # if your cluster supports it, uncomment the following to automatically create
+  # an external load-balanced IP for the frontend service.
+  type: LoadBalancer
+  ports:
+    # the port that this service should serve on
+  - port: 80
+  selector:
+    name: frontend
 ```
 
-Create the service by running:
+#### Using 'type: LoadBalancer' for the frontend service (cloud-provider-specific)
+
+Before you create this frontend service, **you may want to make an edit to this file to set up an external IP for the service**, if this is supported for your cloud provider.
```

Comment:
Doesn't the next paragraph express the same idea? I think we can remove this line.

One more thing, could you change "See the section below on accessing the guestbook site externally for more detail" to more detail**s**